scripts/build-babel/index.js

```diff
@@ -0,0 +1,38 @@
+const babel = require('babel-core');
```

Comment:
script not needed

Reply:
Can you please suggest a good babel plugin to integrate GCC then?

Reply:
i mean the build is literally not using this currently

Reply:
Sorry about the confusion. Way to trigger the build scripts are via node ./scripts/build-babel/index.js and node ./scripts/build-rollup/index.js. I didn't add it to npm scripts. Should have added a note.